### PR TITLE
improve: Add language about 0 block ranges

### PR DESCRIPTION
**Pull request by nicholaspai**
Handles edge case where a chain is paused

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
LGTM!

File: UMIPs/umip-157.md [hunk 1]
```diff
@@ -163,11 +163,14 @@ for the
 [ProposeRootBundle](https://github.com/across-protocol/contracts-v2/blob/a8ab11fef3d15604c46bba6439291432db17e745/contracts/HubPool.sol#L148-L156)
 event that is as late as possible, but earlier than the RootBundleExecuted event we just identified. Once this proposal event is found, determine its
 mapping of indices to `chainId` in its `bundleEvaluationBlockNumbers` array using "CHAIN_ID_LIST". For
-each `chainId`, their starting block number is that chain's `bundleEvaluationBlockNumber + 1` in this previous [valid proposal](#valid-bundle-proposals) event.
+each `chainId`, their starting block number is the minimum of that chain's `bundleEvaluationBlockNumber + 1` in this previous [valid proposal](#valid-bundle-proposals) event and the `latest` block height for the chain.
```
[the remark this answers is not in the document]
Comment (nicholaspai) [reply UMIPs/umip-157.md:166]: is this something we should even handle? That seems like a fundamental assumoption about blockchains that they must increase block height

[the remark this answers is not in the document]
Comment (mrice32) [reply UMIPs/umip-157.md:166]: Not always true with reorgs and/or node providers that have desync between nodes. Either way, I think this is an edge case!

**Review by mrice32 — APPROVED** [review]

File: UMIPs/umip-157.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrice32) [reply UMIPs/umip-157.md:166]: I think it honestly is an edge case, not something we should worry too much about.

**Pull request merged**